src/components/Pagination/index.jsx

```diff
@@ -90,14 +120,20 @@ const Pagination = ({
           </span>
         )}
         <div className="pagination-links">
-          {Array(endPage - startPage + 1)
+          {showPaginationLinks(
+            startPage,
+            endPage,
+            currentPageIndex,
+            updatePageIndex
+          )}
+          {/* {Array(endPage - startPage + 1)
```

Comment:
Can remove this comment now that you abstracted it into the showPaginationLinks function!